docs/providers/aws/events/apigateway.md

```diff
@@ -191,6 +191,41 @@ functions:
             identityValidationExpression: someRegex
 ```
 
+You can also configure an existing Cognito User Pool as the authorizer, as shown
+in the following example:
+
+```yml
+functions:
+  create:
+    handler: posts.create
+    events:
+      - http:
+          path: posts/create
+          method: post
+          authorizer:
+            arn: arn:aws:cognito-idp:us-east-1:xxx:userpool/us-east-1_ZZZ
```

Comment:
that would be great.  Right now I have to set it manually in API Gateway console:

![image](https://cloud.githubusercontent.com/assets/646732/21469997/1dc93e5e-ca29-11e6-8926-2f411d5520d2.png)